src/clib/lib/job_queue/workflow.cpp

```diff
@@ -239,3 +224,8 @@ extern "C" PY_USED const char *
 worflow_get_src_file(const workflow_type *workflow) {
     return workflow->src_file;
 }
+
+ERT_CLIB_SUBMODULE("workflow", m) {
+    m.def("get_last_error",
+          [](Cwrap<config_content_type> self) { return self->parse_errors; });
```

Comment:
I don't really understand why `get_last_error` is returning  `self->parse_errors;` I noticed `script->last_error = content->parse_errors;` but it is still confusing

Reply:
good catch! i think this is copy pasta - while usually delicious, this one clearly has too much salt : / I fix.

Reply:
weird that it didn't get caught by tests :thinking:

Reply:
hm looks like the python method that used the exported C function is only used in the run workflow widget, which at a quick glimpse doesn't seem to be included in our gui tests (really not sure though, as I'm not confident what to grep for)

Reply:
fixed now (the code, nothing done about tests)